test/extended/oauth/client_secret.go

```diff
@@ -107,20 +109,19 @@ var _ = g.Describe("[sig-auth][Feature:OAuthServer]", func() {
 					client := &http.Client{Transport: tr}
 
 					tokenResponse, err = client.Do(tokenReq)
-					// if there was an error then append the error and continue
```

Comment:
you're mixing clauses in a sentence with a conjunctive relation, should still be `if there __was__ an error`